<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SumGenerator</class>
 <widget class="QMainWindow" name="SumGenerator">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>347</width>
    <height>148</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>SumGenerator</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QLabel" name="lblScore">
        <property name="text">
         <string>Score: 0/0</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <widget class="QLineEdit" name="lineEditAnswer"/>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="btnSubmit">
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
    <item row="0" column="0" colspan="2">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="lblSolve">
        <property name="text">
         <string>Solve the following sum: </string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="lblExpression">
        <property name="text">
         <string>TextLabel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="2">
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLabel" name="lblPercent">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>347</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionContext"/>
    <addaction name="actionAbou"/>
   </widget>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Game</string>
    </property>
    <addaction name="actionRating"/>
    <addaction name="actionRestart"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuCredits">
    <property name="title">
     <string>Credits</string>
    </property>
    <addaction name="actionCSS"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
   <addaction name="menuCredits"/>
  </widget>
  <action name="actionContext">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/info.jpg</normaloff>:/img/img/info.jpg</iconset>
   </property>
   <property name="text">
    <string>Context Help</string>
   </property>
  </action>
  <action name="actionAbou">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/about.png</normaloff>:/img/img/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionRestart">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/restart.jpg</normaloff>:/img/img/restart.jpg</iconset>
   </property>
   <property name="text">
    <string>Restart</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/exit.png</normaloff>:/img/img/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionRating">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/rating.png</normaloff>:/img/img/rating.png</iconset>
   </property>
   <property name="text">
    <string>Rating</string>
   </property>
  </action>
  <action name="actionCSS">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/credit.png</normaloff>:/img/img/credit.png</iconset>
   </property>
   <property name="text">
    <string>Styling</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
